lib/cartopy/tests/mpl/test_gridliner.py

```diff
@@ -85,6 +85,16 @@ def test_gridliner():
                         top=1 - delta, bottom=0 + delta)
 
 
+@ImageTesting(['gridliner2'])
+def test_gridliner2():
+    ax = plt.axes(projection=ccrs.PlateCarree())
+    xl = mticker.FixedLocator(range(0, 361, 30))
+    yl = mticker.FixedLocator(range(-90, 91, 10))
+    ax.set_global()
+    ax.coastlines()
+    ax.gridlines(xlocator=xl, ylocator=yl)
```

Comment:
Any chance we could get away without a visual test here? How about checking the gridliner object?